processing/src/main/java/io/druid/segment/serde/ComplexMetricSerde.java

```diff
@@ -36,14 +36,11 @@
    * Deserializes a ByteBuffer and adds it to the ColumnBuilder.  This method allows for the ComplexMetricSerde
    * to implement it's own versioning scheme to allow for changes of binary format in a forward-compatible manner.
    *
-   * The method is also in charge of returning a ColumnPartSerde that knows how to serialize out the object it
-   * added to the builder.
    *
    * @param buffer the buffer to deserialize
    * @param builder ColumnBuilder to add the column to
```

Comment:
Since the return value is not used anywhere, so i removed the return value.
IMO the first line of comments had explained the interface well:
"Deserializes a ByteBuffer and adds it to the ColumnBuilder."